src/Apps/Artwork/Components/TrustSignals/__tests__/AuthenticityCertificate.test.tsx

```diff
@@ -65,4 +70,66 @@ describe("AuthenticityCertificate", () => {
     })
     expect(component.find("TrustSignal").length).toBe(1)
   })
+
+  it("Click on certificate of authenticity link opens modal", async () => {
```

Comment:
My concern with this test is that potentially the expect clauses aren't actually executing. 

Could you add `expect.assertions(2)` on 75 just to make sure everything is being executed as expected?

Reply:
Damn you're right

Reply:
Can we pair on this?